R/redcap_import_recode.R

```diff
@@ -57,7 +57,7 @@ redcap_recode <- function(selected_data,
                     validation = text_validation_type_or_show_slider_number)
 
   # open log-file
-  log_file <- "redcap_recode.txt"
+  log_file <- "redcap_import_recode.txt"
```

Comment:
maybe you want to give the user control over the name/location of the log file via an option to the function? (could be that they need to recode multiple datasets...)

Reply:
i did not make changes for this function (besides the name-change that you suggested)